tests/StackExchange.Redis.Tests/BacklogTests.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace StackExchange.Redis.Tests
+{
+    public class BacklogTests : TestBase
+    {
```

Comment:
Is there a good wait to test that handshake commands fail fast, even with the default BacklogPolicy?

Reply:
Since handshake commands won't go through this they shouldn't be subject to the policy in general - at least that's the intent, but if you see otherwise I need to fix!